docs/develop-load-manager.md

```diff
@@ -207,3 +103,216 @@ then all machines are approximately overloaded. In the case in which a broker's
 threshold, that broker is not considered for bundle assignment. If all brokers are overloaded, the bundle is randomly
 assigned.
 
+## Enablement
+
+You can enable a simple, modular, or extensible load manager by following the steps below.
+
+### Enable simple load manager
+
+You can enable the simple load manager using one of the following methods: 
+
+- Method 1
+
+  Update the value of [loadManagerClassName](https://github.com/apache/pulsar/blob/782e91fe327efe2c9c9107d6c679c2837d43935b/conf/broker.conf#L1309) to `org.apache.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl` in `conf/broker.conf`.
+
+- Method 2
+  
+  Use the `pulsar-admin` tool. 
+
+   ```shell
+   pulsar-admin brokers update-dynamic-config \
+   --config loadManagerClassName \
+   --value org.apache.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl
+   ``` 
+
+  :::note 
+  
+  Any mistakes in specifying the load manager will cause Pulsar to default to `SimpleLoadManagerImpl`.
+
+  :::
+
+### Enable modular load manager
+
+You can enable the modular load manager using one of the following methods: 
+
+- Method 1
+
+  Update the value of [loadManagerClassName](https://github.com/apache/pulsar/blob/782e91fe327efe2c9c9107d6c679c2837d43935b/conf/broker.conf#L1309) to `org.apache.pulsar.broker.loadbalance.impl.ModularLoadManagerImpl` in `conf/broker.conf`.
+
+- Method 2
+  
+  Use the `pulsar-admin` tool. 
+
+   ```shell
+   pulsar-admin brokers update-dynamic-config \
+   --config loadManagerClassName \
+   --value org.apache.pulsar.broker.loadbalance.impl.ModularLoadManagerImpl
+   ```
+
+  :::note 
+  
+  Any mistakes in specifying the load manager will cause Pulsar to default to `SimpleLoadManagerImpl`.
+
+  :::
+
+### Enable extensible load manager
+
+You can enable the extensible load manager by updating the value of [loadManagerClassName](https://github.com/apache/pulsar/blob/782e91fe327efe2c9c9107d6c679c2837d43935b/conf/broker.conf#L1309) to `org.apache.pulsar.broker.loadbalance.impl.ExtensibleLoadManagerImpl` in `conf/broker.conf`.
+
+:::note
+
+The pulsar-admin tool is not supported for enabling the extensible load manager.
+
+:::
```

Comment:
@Demogorgon314 is this note correct here? As same to https://github.com/apache/pulsar-site/pull/657/files#diff-2abdb541a434c7250793845da41718b322344e45cc83098c3c57addacc4b5a6aR83-R85

Reply:
Yes, is corrected.